src/python_testing/TC_WEBRTC_1_2.py

```diff
@@ -169,7 +172,7 @@ async def test_TC_WEBRTC_1_2(self):
         ice_session_id, remote_candidates = await webrtc_peer.get_remote_ice_candidates()
         asserts.assert_equal(prev_sessionid, ice_session_id, "ProvideIceCandidates invoked with wrong session id")
         asserts.assert_true(len(remote_candidates) > 0, "Invalid remote ice candidates received")
-        webrtc_peer.set_remote_ice_candidates(remote_candidates)
+        webrtc_peer.set_remote_ice_candidates([cand.candidate for cand in remote_candidates])
```

Comment:
Looking at the REPL failure and the API, remote_candidates is a list[str] and so isn't iterable in this way. Also set_remote_ice_candidates expects a list[str], so I'm not sure why this change?

Reply:
Thanks for pointing it out. The new API signature update is missed here as you mentioned. Updated in the latest PS.